docs/resources.rst

```diff
@@ -15,6 +15,11 @@ General
 - `Vyper Hub for development <https://github.com/zcor/vyper-dev>`__
 - `Vyper greatest hits smart contract examples <https://github.com/pynchmeister/vyper-greatest-hits/tree/main/contracts>`__
 
+Getting started
```

Comment:
Not sure this needs a whole new section? Would it make sense for this to go in the previous section?